<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F2EC;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../img/logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>22</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">color: #356a9c;
font: 20pt &quot;Bahnschrift&quot;;</string>
               </property>
               <property name="text">
                <string>Create an account</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
            </property>
            <property name="text">
             <string>Full name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtFullName">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
            </property>
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtEmail">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #575556;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #ECE3D2;
color: #356A9C;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 15px;
padding-left: 5px;</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Bahnschrift&quot;;
color: #575556;</string>
            </property>
            <property name="text">
             <string>I have read and agree to AdoptMate's privacy Policy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRegister">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 15px;
font: 600 16pt &quot;Bahnschrift&quot;;
background-color: #356a9c;
color: #F8F2EC;

</string>
            </property>
            <property name="text">
             <string>Sign Up</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_login">
            <property name="minimumSize">
             <size>
              <width>52</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: #356a9c;
background-color: rgba(255, 0, 0, 0);
</string>
            </property>
            <property name="text">
             <string>Already have an account?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-radius: 15px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>400</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../img/3766277.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
